searxng/settings-base.yml

```
@@ -0,0 +1,15 @@
+use_default_settings: true
+server:
+  # base_url is defined in the SEARXNG_BASE_URL environment variable, see .env and docker-compose.yml
+  secret_key: "ultrasecretkey"  # change this!
+  limiter: false
+  image_proxy: true
+ui:
+  static_use_hash: true
+redis:
+  url: redis://redis:6379/0
+search:
+    formats:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)